packages/component-library/components/Dropdown/Dropdown.module.scss

```
@@ -6,7 +6,7 @@
 
 $width: 248px;
 $caButton-border-width: $border-solid-border-width;
-$caButton-verticalPadding: calc(#{$ca-grid / 2} - #{$caButton-border-width});
+$caButton-verticalPadding: #{calc($ca-grid / 2)} - #{$caButton-border-width};
```

```suggestion
$caButton-verticalPadding: calc($ca-grid / 2) - #{$caButton-border-width};
```
This syntax might be confusing me, but I don't think you need to interpolate the whole equation. Unsure!